app/controllers/production_releases_controller.rb

```diff
@@ -0,0 +1,37 @@
+class ProductionReleasesController < SignedInApplicationController
+  before_action :require_write_access!
+  before_action :set_release_platform_run, only: %i[create]
+  before_action :set_prod_release, only: %i[changes_since_previous]
+  before_action :set_app, only: %i[changes_since_previous]
+
+  def create
```

Comment:
ensure the release platform run is active. either here or in Coordinator action.